.travis.yml

```diff
@@ -68,7 +69,7 @@ env:
 
 before_install:
   - unset SBT_OPTS JVM_OPTS
-  - docker pull ziemin/pio-testing
+  - docker pull chanlee514/pio-testing
```

Comment:
I think we should put this under `predictionio` org on Docker Hub. We are missing something that automatically build and tag these testing container images properly. Let me work on it and we can modify our integration test to pick up the proper container.